.mailmap

```diff
@@ -0,0 +1,50 @@
+Anibal Sólon Heinsfeld <anibalsolon@gmail.com>
+Christopher J. Markiewicz <effigies@gmail.com>
+Christopher J. Markiewicz <effigies@gmail.com> <effigies@bu.edu>
+Christopher J. Markiewicz <effigies@gmail.com> <markiewicz@stanford.edu>
+Daniel J. Lurie <danjlurie@gmail.com>
+Daniel J. Lurie <danjlurie@gmail.com> Daniel Lurie <danjlurie@gmail.com>
+Daniel J. Lurie <danjlurie@gmail.com> danlurie <danjlurie@gmail.com>
+Elizabeth DuPre <emd222@cornell.edu>
+Franklin Feingold <ffein@stanford.edu>
+Franklin Feingold <ffein@stanford.edu> Franklin Feingolr <35307458+franklin-feingold@users.noreply.github.com>
```

Comment:
Here, because it's a new email, you need to map it. But there's no advantage to adding the name as well unless you need to catch the case where two people used the same email but different names (e.g., `<(nobody)@(none)>`.)

```suggestion
Franklin Feingold <ffein@stanford.edu> <35307458+franklin-feingold@users.noreply.github.com>
```